### PR TITLE
bringing API up to current spec

### PR DESCRIPTION
**Pull request by eropple**

### CODE REVIEW

**Review by jaredsmith — APPROVED** [review]
This all looks good and straightforward to me.

File: src/video/resources/playbackRestrictions.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+} from '../domain.js';
+
+/**
+ * @private Base playback ID path for the Mux API
+ * */
+const PATH = '/video/v1/playback-restrictions';
+
+/**
+ * @private
+ * Build the base playback ID path for the Mux API
```
Comment (jsanford8) [inline src/video/resources/playbackRestrictions.ts:16]: I think you don't mean playback ID here in the comment, right?

**Review by jsanford8 — APPROVED** [review]
two comments, one a typo in a comment and one question about ts but I think this is good to go once those are changed

File: src/video/domain.ts [hunk 2]
```diff
@@ -281,3 +294,46 @@ export interface ListUploadParams {
...
+
+export interface GetLiveStreamPlaybackIdResponse {
+  data: PlaybackId;
+}
+
+export interface LiveStreamEmbeddedSubtitleSettings {
+  name: string;
+  passthrough?: string;
+  language_code?: string;
+  language_channel?: 'cc1';
```
[the remark this answers is not in the document]
Comment (eropple) [reply src/video/domain.ts:310]: That is correct - it's either omitted or `cc1`.

File: src/video/resources/playbackRestrictions.ts (hunk 1 above)
Comment (eropple) [reply src/video/resources/playbackRestrictions.ts:16]: woops. correct

**Pull request merged**